contents/swift/welcome-to-swift.md

```diff
@@ -12,44 +12,46 @@
+Authors: [Ch'ng Ming Shin](https://github.com/ablyx/cs3281-website/blob/mingshin-week6-progress/students/AY1617S2/ch'ngMingShin/Ch'ngMingShin-Resume.md), [Yong Zhi Yuan](https://github.com/Zhiyuan-Amos), [Jiang Chunhui](https://github.com/Adoby7)
 
-# Overview
+## What is Swift?
 
-Welcome to Swift, THE preferred language to do iOS programming. Introduced in 2014, Swift is a fast, modern and safe programming language which supports playgrounds, an innovative feature that allows programmers to experiment with Swift code and see the results immediately, without the overhead of building and running an app.
+Swift is a fast, modern and safe programming language mainly used for iOS programming.
+Introduced in 2014, Swift has concise and expressive syntax, compared with its predecessor language **Objective-C**. It also supports playgrounds, an innovative feature that allows programmers to experiment with Swift code and see the results immediately, without the overhead of building and running an app.
 
-# Getting Started
+## Why should I learn Swift?
 
-Reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is definitely recommended, since it explains everything quite clearly, albeit quite verbose.
+1. **iOS developers have good employment.** Mobile development becomes popular recently. A [survey in StackOverFlow](https://insights.stackoverflow.com/survey/2017#developer-profile-developer-type) shows that mobile development, although is significantly less than web development, still occupies large marketing along with desktop application development. In addition, iOS developer is well-paid. The [salary statistics](https://www.indeed.com/salaries/iOS-Developer-Salaries) shows that iOS developer in US can earn more than $120K per year. Therefore, learning Swift is helpful for your career.
 
-If you're just looking to take a dip into Swift, here are the [basics](https://guides.codepath.com/ios/Swift-Basics) you'll need to understand.
+1. **Swift is easy to learn.** Swift is aimed to be designed as a simple and understandable language by Apple. Its syntax resembles English, and thus is easy to read and understand. Any programmer being familiar with Python, Java or C++ can learn Swift quickly. With its simplistic and readable syntax, learning Swift is easy even for those just starting out with programming!
 
-And if you are really pressed for time, here's a couple of cheatsheets with code examples:
+1. **Swift is an [open source project](https://github.com/apple/swift).** Learning about Swift and contributing to the language may be a useful experience for those interested. In addition, its source code may be useful reading for those interested in learning good practices, or simply to find out more about how the language was implemented.
```

Comment:
Try something like this?

Apart from learning about Swift from its documentation, one can also dig into the source code to get new insights. Being open-source means you can also start submitting issues or patches if you find there is any way to improve the language!